tests/WithDb/SelectTest.php

```diff
@@ -17,7 +17,7 @@ protected function setUp(): void
         $this->c = Connection::connect($GLOBALS['DB_DSN'], $GLOBALS['DB_USER'], $GLOBALS['DB_PASSWD']);
 
         $pdo = $this->c->connection();
-        $pdo->query('CREATE TEMPORARY TABLE employee (id int not null, name text, surname text, retired bool, PRIMARY KEY (id))');
+        $pdo->query('CREATE TABLE employee (id int not null, name text, surname text, retired bool, PRIMARY KEY (id))');
```

Comment:
@DarkSide666 hope only for tests ;-)

Reply:
yes just testing because I do not have pgsql on my local ready for tests